word/views_api.py

```diff
@@ -0,0 +1,42 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from fuzzywuzzy import fuzz
+from fuzzywuzzy import process
+from .models import Word, Description
+
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
+
+
+@csrf_exempt
+@api_view(['GET'])
+@authentication_classes([])
+@permission_classes([])
+def similar_words(request):
+    if request.GET:
+        base_word = request.GET.get("word")
+        if base_word:
+            list_of_words = list(Word.objects.all().values_list("word", flat=True))
+            # fuzzywuzzy: Get a list of matches ordered by score, the top 3
+            results = process.extract(base_word, list_of_words)[0:3]
+            list_of_similar_words = [x[0] for x in results]
+            words_desc_ids = []
+
+            for word in list_of_similar_words:
+                word_obj = Word.objects.filter(word=word).first()
```

Comment:
We have to rethink this, just filtering  by the word is not enough. It needs to get the exact word it was closest to. For example if the three closest words to "lite" were "light" (opposite of heavy), "light" (as in flashlight) and "line", then it will get one of the "light" words twice. So objects.get(id) is better